instructure/canvas-ios
# Finish Parent QR account creation

refs: MBL-14413
affects: parent, student
release note: Added pairing with students from the launch screen

[run-nightly]

Test plan:
- Log into Parent and enable qrLoginParent and parentQRCodePairing
- Log out of Parent
- Tap the QR button
- Tap 'I don't have a Canvas account'
- Download Student app and enable `studentQRCodePairing`
- Settings -> Pair with Observer
- Scan the QR code from the Parent app
- You should be taken to the Create Account screen with the pairing code set
- Create a new account and tap Start Participating
- You should be logged in and paired with the user
- Log in as the same parent on web
- Log out of the Parent app
- Tap the QR button
- Tap 'I have a Canvas account'
- Scan the QR code from canvas web to login
- You should be logged in
- Tap add Student from the Parent dashboard
- Use the QR scanner to add a second child
